### PR TITLE
Changes class fields to private and adds getter and setter methods

### LINKED ISSUE #51: member variables should not be public
### Description
use accessors/mutators when appropriate

### PR DESCRIPTION
**Pull request by akhila-raju**
## PR description
Instead of making class fields public, I switched them to private and added getter() and setter() methods for these class fields.

**Note:** Commented out lines that were causing a build error as captured in #56.

## Fixed Issue(s)

Fixes #51.

### CODE REVIEW

File: artemis/src/main/java/tech/pegasys/artemis/datastructures/BeaconChainState/CandidatePoWReceiptRootRecord.java [hunk 1]
```diff
@@ -18,12 +18,12 @@
 
 public class CandidatePoWReceiptRootRecord {
 
-
   private Hash candidate_pow_receipt_root;
   private UInt64 votes;
```
Comment (jrhea) [inline artemis/src/main/java/tech/pegasys/artemis/datastructures/BeaconChainState/CandidatePoWReceiptRootRecord.java]: Did you want to make getters and setters for these member variables?

File: pow/src/main/resources/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Truper is a utility to compile Vyper contracts for Truffle
 
 `npm i -g truper`
 
-`cd vrc`
+`cd pom`
```
Comment (jrhea) [inline pow/src/main/resources/README.md]: should this be:
`cd pow` ?

Comment (akhila-raju) [reply pow/src/main/resources/README.md]: I think so, right? Isn't this vrc folder the old one? + @dangerousfood

Comment (akhila-raju) [reply pow/src/main/resources/README.md]: Just realized I put pom and not pow 😂Thanks for fixing!

**Pull request merged**